lib/src/runner/engine.dart

```diff
@@ -340,8 +340,11 @@ class Engine {
   ///
   /// If [countSuccess] is `true` (the default), the test is put into [passed]
   /// if it succeeds. Otherwise, it's removed from [liveTests] entirely.
+  ///
+  /// [attempt] is used to determine if this test should be retried, defaults to
+  /// 1.
   Future _runLiveTest(LiveSuiteController suiteController, LiveTest liveTest,
-      {bool countSuccess: true}) async {
+      {bool countSuccess: true, int attempt: 1}) async {
```

Comment:
We generally avoid setting default values for non-boolean arguments, since it makes it difficult for callers to explicitly specify "the default value". Even in cases like this, where no callers are doing that, I like to use `??` in the body for consistency.